cfgov/jinja2/v1/_includes/organisms/ask-search.html

```diff
@@ -54,7 +57,21 @@
                                name="q"
                                id="o-search-bar_query"
                                value="{{ ask_query }}"
-                               placeholder="{{ placeholder }}">
+                               placeholder="{{ placeholder }}"
+                               maxlength="{{ max_length }}"
+                               aria-describedby="o-search-bar-error_message">
+                    </div>
+                    {# TODO: Is there a better way to render this hidden initially? #}
```

Comment:
I think this is fine. This is a patten we use elsewhere. Have `u-hidden` in markup and remove using JS.

Reply:
Works for me. I removed the TODO in 8b4c07f.